pkg/kv/export/export_test.go

```diff
@@ -0,0 +1,48 @@
+package export_test
+
+import (
+	"context"
+	_ "embed"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/pkg/kv/export"
+	"github.com/treeverse/lakefs/pkg/kv/kvtest"
+	_ "github.com/treeverse/lakefs/pkg/kv/mem"
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+func TestImport(t *testing.T) {
+	ctx := context.Background()
+	logger := logging.Default()
+	store := kvtest.MakeStoreByName("mem", "")(t, context.Background())
```

Comment:
Calling this function returns a factory - you can keep a reference
```
kvtest.MakeStoreByName("mem", "")
```
There is no need to create a factory every time you need a storage - you can even add a different package to create the test storage.
So in case we like to test using a different implementation we can switch in a single place